CONTRIBUTING.md

```diff
@@ -1,3 +1,34 @@
+# Contributing to Elasticsearch Ruby
+
+To work on the code, clone and bootstrap the project first:
+
+```
+git clone https://github.com/elasticsearch/elasticsearch-ruby.git
+cd elasticsearch-ruby/
+bundle exec rake bundle
+```
+
+This will run `bundle install` in all subprojects. 
+
+To run the tests, you need to start a testing cluster on port 9250, or provide a different one in the `TEST_CLUSTER_PORT` environment variable.
+
+There's a Rake task to start a testing cluster in a Docker container:
+
+`rake docker:start[version]` - E.g.: `rake docker:start[7.x-SNAPSHOT]`. To start the container with Platinum, pass it in as a parameter: `rake docker:start[7.x-SNAPSHOT,xpack]`.
+
+
+To run all the tests in all the subprojects, use the Rake task:
+
+```
+time rake test:client
+```
+
+By default, tests will atempt to use `http://localhost:9200` as a test server. If you're using a different host/port, set the `TEST_ES_SERVER` environment variable, e.g.:
```

Comment:
Might be worth explicitly mentioning that it is the responsibility of the developer to provide a test server. It is outside the scope of this project to do so, although Docker is recommended!

Reply:
I rewrote this part, let me know what you think

Reply:
Perfect